app/Http/Controllers/BookController.php

```diff
@@ -98,6 +105,7 @@ public function store(Request $request, string $shelfSlug = null)
             'description' => ['string', 'max:1000'],
             'image'       => array_merge(['nullable'], $this->getImageValidationRules()),
             'tags'        => ['array'],
+            'default_template'  => ['nullable', 'exists:pages,id'], 
```

Comment:
Right now there's nothing stopping a user from setting a page template they don't have access to. 
We probably should have some light validation to check the user can view and that the template page is marked as a template. 

This exists validation check could also be abused to check the existence of a certain page id what the user may not have access to. Not sure how well we currently defend against that in other areas of the app, but probably would be something I'd attempt to avoid where possible.